pyquil/tests/test_quantum_computer.py

```diff
@@ -178,6 +311,17 @@ def test_readout_symmetrization(forest):
     assert diff_s < 0.05
 
 
+def test_run_symmetrized_readout_error():
+    # This test checks if the function runs for any possible input on a small number of qubits.
+    # Locally this test was run all 8 qubits, but it was slow.
+    qc = get_qc("8q-qvm")
```

Comment:
let's use a fixture. if there's not already one for a QVM-backed QC object, we should make one.

Reply:
I'm pretty [green behind the ears](https://languagelog.ldc.upenn.edu/nll/?p=727) so I don't know what a fixture is or what it means to have a QVM-backed QC object, which means I don't understand what you are asking me to do. Can you help me with this one?